<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="altSaveAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="allSelectAction"/>
    <addaction name="separator"/>
    <addaction name="fontAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_R"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newFileAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openFileAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="altSaveAction">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>撤销(&amp;U)
</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>重做(&amp;R)
</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切(&amp;X)
</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制(&amp;C)
</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴(&amp;V)
</string>
   </property>
  </action>
  <action name="allSelectAction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
